README.md

```diff
@@ -10,15 +10,37 @@ Sporkfish is a Python-based chess engine. Chess programming techniques, although
 
 ### Set up
 
-To quickly set up the environment, you can use docker. From a terminal in the root directory:
+As a general case `Using DevContainer with VSCode` or `Using Github Codespace`
```

Comment:
Could we change this to: 

We can quickly setup the environment by utilizing DevContainer in VSCode, employing GitHub Codespaces or using naive docker (or somethin similar). Then give the 3 options?